tests/Integration/bootstrap.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Bootstraps the WP Rocket Plugin integration tests
```

Comment:
```suggestion
 * Bootstraps the Imagify Plugin integration tests
```

I see what you did here.